package.json

```diff
@@ -19,13 +20,14 @@
   },
   "homepage": "https://github.com/LateralView/spotify-exercise#readme",
   "dependencies": {
-    "angular": "^1.5.8",
-    "angular-ui-router": "^0.3.2",
+    "angular": "^1.6.0",
+    "angular-ui-router": "^1.0.0-beta.3",
```

Comment:
We don't normally use beta versions, because we know that this beta allow us to do the things in a better way (using the component hook) but if we reach production before this got working we will have to roll it back (on a real case)